docs/resources/compute_acl.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: "fastly"
+page_title: "Fastly: compute_acl"
+sidebar_current: "docs-fastly-resource-compute_acl"
+description: |-
+  Provides a Compute Access Control List (ACL) that you can attach to a Compute service.
+---
+
+# fastly_compute_acl
+
+Provides a Compute Access Control List (ACL) that can be linked to a Fastly Compute service. ACLs define CIDR-based access rules (e.g., allow/block IP ranges).
+
+To make a `fastly_compute_acl` available to a [Compute](https://developer.fastly.com/learning/compute/) service, you must create a `fastly_service_compute` resource and attach the ACL using a `resource_link` block, as shown below.
+
+## Example Usage
+
+Basic usage:
+
+```terraform
+# IMPORTANT: Deleting a Compute ACL requires first deleting its resource_link.
+# This requires a two-step `terraform apply` because we can't guarantee deletion order.
```

Comment:
Yuck... this would work properly if `resource_link` was a resource of its own instead of nested. Another thing to improve when we build a new provider :-)